cypress/locators/split-button/index.js

```diff
@@ -0,0 +1,4 @@
+import { TOGGLE_BUTTON } from './locators';
```

Comment:
Why locator is called toggle_button, if we have split_button component ?

Reply:
data element is called toggle-button

Reply:
fixed to splitToggleButton and SPLIT_TOGGLE_BUTTON